components/pipedrive/actions/add-activity/add-activity.mjs

```diff
@@ -1,133 +1,154 @@
       optional: true,
     },
     duration: {
       type: "string",
+      label: "Duration",
       description: "Duration of the activity. Format: HH:MM",
       optional: true,
     },
-    user_id: {
-      type: "integer",
+    userId: {
+      propDefinition: [
+        pipedriveApp,
+        "userId",
+      ],
       description: "ID of the user whom the activity will be assigned to. If omitted, the activity will be assigned to the authorized user.",
-      optional: true,
     },
-    deal_id: {
+    dealId: {
       type: "string",
+      label: "Deal ID",
       description: "ID of the deal this activity will be associated with",
       optional: true,
     },
-    person_id: {
-      type: "integer",
+    personId: {
+      propDefinition: [
+        pipedriveApp,
+        "personId",
+      ],
       description: "ID of the person this activity will be associated with",
-      optional: true,
     },
     participants: {
       type: "any",
+      label: "Participants",
       description: "List of multiple persons (participants) this activity will be associated with. If omitted, single participant from person_id field is used. It requires a structure as follows: [{\"person_id\":1,\"primary_flag\":true}]",
       optional: true,
     },
-    org_id: {
-      type: "integer",
+    organizationId: {
+      propDefinition: [
+        pipedriveApp,
+        "organizationId",
+      ],
       description: "ID of the organization this activity will be associated with",
-      optional: true,
     },
     note: {
       type: "string",
+      label: "Note",
       description: "Note of the activity (HTML format)",
       optional: true,
     },
     location: {
       type: "string",
+      label: "Location",
       description: "The address of the activity. Pipedrive will automatically check if the location matches a geo-location on Google maps.",
       optional: true,
     },
-    public_description: {
+    publicDescription: {
       type: "string",
+      label: "Public Description",
       description: "Additional details about the activity that will be synced to your external calendar. Unlike the note added to the activity, the description will be publicly visible to any guests added to the activity.",
       optional: true,
     },
-    busy_flag: {
+    busyFlag: {
       type: "boolean",
+      label: "Busy Flag",
       description: "Set the activity as 'Busy' or 'Free'. If the flag is set to true, your customers will not be able to book that time slot through any Scheduler links",
       optional: true,
     },
     attendees: {
       type: "any",
+      label: "Attendees",
       description: "Attendees of the activity. This can be either your existing Pipedrive contacts or an external email address. It requires a structure as follows: [{\"email_address\":\"mail@example.org\"}] or [{\"person_id\":1, \"email_address\":\"mail@example.org\"}",
       optional: true,
     },
```

Comment:
Could we use async options for `attendees`? They'd be the same as the options for `userId`, but with the email as the value.